src/main/java/seedu/address/model/Model.java

```diff
@@ -147,7 +148,7 @@ void updatePerson(Person target, Person editedPerson)
     /**
      * Sorts the pet list.
      */
-    void sortPetList();
+    void sortPetList() throws ClientPetAssociationListEmptyException;
```

Comment:
Maybe header comments can be more specific about how its sorted.

Reply:
alright will issue the amendments by tonight! 👍 thanks!

Reply:
resolved